terraform/main.tf

```
@@ -107,7 +107,7 @@ data "template_file" "kali_userdata" {
     hostname        = "kali-pentest-01"
     ssh-pub-key-1   = "${local.ssh-pub-key-1}"
     ssh-pub-key-2   = "${local.ssh-pub-key-2}"
```

Could this be a list instead? 
```suggestion
    ssh-keys  = ["${local.ssh-pub-key-1}", "${local.ssh-pub-key-2}"]
```